builder/googlecompute/config.go

```diff
@@ -366,6 +375,15 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 
 	var errs *packersdk.MultiError
 
+	for i, bd := range c.ExtraBlockDevices {
+		err := bd.Prepare()
+		if err != nil {
+			errs = packersdk.MultiErrorAppend(errs, err...)
```

Comment:
If bd.Prepare fails should we skip the setting of bd.zone and c.ExtraBlockDeviecs? 
```suggestion
			errs = packersdk.MultiErrorAppend(errs, err...)
			continue
```

Reply:
Good call, we definitely can